ocs_ci/ocs/node.py

```
@@ -55,6 +57,149 @@
+
+        Returns:
+            str: Output of the command execution
+
+        Raises:
+            CommandFailed: In case the command execution fails
+
+        """
+        namespace = namespace or self.namespace
+        use_root = use_root if use_root is not None else self.use_root
+
+        log.info(f"Running command '{cmd}' on node '{self.resource_name}'")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)